### ISSUE TITLE
[Suggestion]: More Sub-Phases to make it feel smoother.

### ISSUE THREAD

**Issue by NinjaSam5399** [nonamecrackers2/crackers-wither-storm-mod#1453]
### Description of feature request

I know that you confirmed that there will be more subphases in between phase 3-4 (#1431). But, I was gonna plan for more Wither Storm Sub-phases that other than phase 3-4 so it will feel like the wither storm is actually growing.
List of Subphases:
- Phase 2.2: Looks similar to the old version of phase 2 except the wounds near the command block and the tail are a difference between the old one.
- Phase 2.3: The body will almost cover the command block.
- Phase 2.4: The command block are a little bit visible now.
- Phase 2.5: The command block is completely covered but the tentacles still do not appear until the beginning of phase 5.
- Phase 3.3: #1431 
- Phase 3.4: ^
- Phase 3.5: ^
- Phase 3.6: ^
- Phase 3.7: ^
- Phase 3.8: ^
- Phase 3.9: ^ (Finally)
- Phase 4.25: (Idk how to explain it) but it almost looks similar to phase 4.0 and 4.5.
- Phase 5.75: its head will look alot more bumpier and a bit bigger.
- Phase 7.3: The white flickering block will start appearing but the wound that are enterable are not visible.
- Phase 7.4: The Wound will start appearing, but it is not big enough to be entered, so it would be like the wound in the middle of the big head of the wither storm.

Since its a lot, you could remove some sub-phases, or just risk doing all of this list of subphase.

**Comment by NinjaSam5399** [comment]
Since it is a huge transition between phase 3-4, it would need 9 sub-phases.

**Comment by TaintedBlock** [comment]
This is def not happening lmao. And more subphases building up to 4 will come

**Comment by NinjaSam5399** [comment]
> This is def not happening lmao. And more subphases building up to 4 will come

I just said, he could remove some subphases.

**Comment by NinjaSam5399** [comment]
He would probably add 5 subphase in phase 3 instead of 10.

**Comment by TaintedBlock** [comment]
> He would probably add 5 subphase in phase 3 instead of 10.

Again, subphases are already planned for 3. And Nazaru wont want to make a whole bunch more models and change the model system again. The subphases I feel are fine how they are.

**Comment by NinjaSam5399** [comment]
ok

**Comment by nonamecrackers2** [comment]
We don't plan on adding more sub phases to phase 4 and up. I'm not sure about the hunchback sub phases since @Thefuzz1234555 are in charge of those, but I believe they said they're planning on adding more (to which phase(s) I'm not sure).

**Comment by NinjaSam5399** [comment]
> We don't plan on adding more sub phases to phase 4 and up. I'm not sure about the hunchback sub phases since @Thefuzz1234555 are in charge of those, but I believe they said they're planning on adding more (to which phase(s) I'm not sure).
ok

**Comment by mythicplays** [comment]
The Phase 5-5.5 needs more subphases though, phase 5.1, consumedentities around 220k, phase 5.2, 250k, phase 5.25 should be changed to 5.3 with the same consumedentities as phase 5.25, phase 5.4 will be around 320k consumedentities

**Comment by TaintedBlockmObile** [comment]
> The Phase 5-5.5 needs more subphases though, the Phase 6-6.5 should also be smaller too and have 2 more subphases too. Phase 6.2(a bit bigger phase 6) and Phase 6.3(3 heads per each body for 9 heads but same size as phase 6.2)

I disagree with this one. 5-5.5 is fine due to it have 5.25 in the middle. 6-6.5 is alright and there is a way to make 6.5 have e heads with the ripped body as well. And the difference between 6.5 and 7 is not very large at all so having these super small subphases for big phases wouldn't be needed.

**Comment by BluePineappleGuy** [comment]
just wanted to mention that stage 6.5 should look exactly like stage 6, only with 3 heads, as of current its mass grows which makes it look like stage 7

**Comment by TaintedBlockmObile** [comment]
> just wanted to mention that stage 6.5 should look exactly like stage 6, only with 3 heads, as of current its mass grows which makes it look like stage 7

If you decrease phase 6.5's consumed entities this phase can be achieved

**Comment by mythicplays** [comment]
the phase 5-5.5 needs subphases though, also the first tentacle should appear between phase 2.5 and 2.7 and the phase 3 should be a bit bigger. As for phase 6, I did decrease the consumed entities for phase 6.5 and got the phase 6 size with 3 wither storm heads each body, the phase 6 should be smaller too so that it looks like wither storm actually was formidibombed. Also there should be Phase 5.75(mentioned in this suggestion) that will be wider and taller than Phase 7 without the segments(comsumedentities around 450k-500k)

**Comment by mythicplays** [comment]
> > The Phase 5-5.5 needs more subphases though, phase 5.1, consumedentities around 220k, phase 5.2, 250k, phase 5.25 should be changed to 5.3 with the same consumedentities as phase 5.25, phase 5.4 will be around 320k consumedentities
> 
> I disagree with this one. 5-5.5 is fine due to it have 5.25 in the middle. 6-6.5 is alright and there is a way to make 6.5 have e heads with the ripped body as well. And the difference between 6.5 and 7 is not very large at all so having these super small subphases for big phases wouldn't be needed.

**Comment by Steve982950291** [comment]
#1431 Duplicate.
